kview/src/main/java/dev/ikm/komet/kview/events/pattern/PatternFieldsPanelEvent.java

```diff
@@ -23,6 +23,10 @@ public class PatternFieldsPanelEvent extends Evt {
 
     public static final EvtType<PatternFieldsPanelEvent> PATTERN_FIELDS = new EvtType<>(Evt.ANY, "PATTERN_FIELDS");
 
+    public static final EvtType<PatternFieldsPanelEvent> EDIT_FIELDS = new EvtType<>(Evt.ANY, "EDIT_FIELDS");
```

Comment:
FYI,
Event design allows you to have a event type hierarchy assuming that is what you want to represent. Since I'm not familiar with the flow I'm assuming EDIT_FIELDS and ADD_FIELDS is a type of PATTERN_FIELDS. And if so, can be like the following:
```java
public static final EvtType<PatternFieldsPanelEvent> EDIT_FIELDS = new EvtType<>(PATTERN_FIELDS, "EDIT_FIELDS");
```
You may want to check with @dholubek

Reply:
Will check with Dough.